package.json

```diff
@@ -3,7 +3,8 @@
 	"version": "1.0.0",
 	"description": "Generated by cds-services-archetype",
 	"devDependencies": {
-		"@sap/cds-dk": "^8"
+		"@sap/cds-dk": "^9",
+		"@sap/cds-mtxs": "^3"
 	},
 	"workspaces": [
```

Comment:
@beckermarc Shall I remove the workspace or keep ?

Reply:
We should keep it, as this is still the recommended setup and the `@sap/cds-mtxs` dependency here will hopefully vanish again. If we know that it will not vanish, we can remove the workspace later.

Reply:
ok, will keep it.